truss/remote/baseten/utils/status.py

```diff
@@ -0,0 +1,29 @@
+STATUS_TO_DISPLAYABLE = {
```

Comment:
this is temporary translation -- once we have a REST API for chains, we can start using that and then don't need to do this translation on the client